File: Reserva-Hotel-com-Front-main/ProjetoCs-Reserva_de_Hotel-main/ReservaDeHotel/Models/Comodidade.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ReservaDeHotel.Models;
public class Comodidade
{
    [Key]
    public int? IDComodidade { get; set; }
    public int? NumeroDoQuarto { get; set; } 
    public string? TipoDeQuarto { get; set; }
    public decimal? PrecoPorNoite { get; set; }
    public string? Descricao { get; set; }
    public bool? Disponibilidade { get; set; }
 
}

File: Reserva-Hotel-com-Front-main/ReservaHotelFront/src/app/Comodidade.ts
import { Hotel } from './Hotel';

export class Comodidade {
    iDComodidade: number = 0;
    numeroDoQuarto: number = 0;
    tipoDeQuarto: string = '';
    precoPorNoite: number = 0;
    descricao: string = '';
    disponibilidade: boolean | undefined;
    hotelId: number = 0;
    hotel: Hotel | undefined;
}
